packages/components/src/util/URLResolver.ts

```diff
@@ -327,6 +328,17 @@ export class URLResolver {
                 }
             }),
 
+            new ActionMapper('query', 'executeQuery', row => {
```

Comment:
This mapper should return `false` to avoid unnecessary warning of not being able to map other variants of this URL (since it's very common). Additionally, I'd recommend this being even more defensive and checking the for the filter parameter as well.

```ts
new ActionMapper('query', 'executeQuery', row => {
    const url = row.get('url');
    if (url) {
        const materialIdKey = 'query.MaterialId~eq';
        const params = ActionURL.getParameters(url);
        if (
            params &&
            params.schemaName &&
            params.schemaName.toLowerCase() === 'inventory' &&
            params.queryName &&
            params.queryName.toLowerCase() === 'item' &&
            params[materialIdKey] !== undefined
        ) {
            return createProductUrl(
                'freezerManager',
                undefined,
                AppURL.create('rd', 'sampleItem', params[materialIdKey])
            );
        }
    }

    return false;
}),
```